CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.10.31-dev1
+
+### Enhancements
+
+### Features
+
+### Fixes
+* **Remove default user ./ssh folder** The default notebook user during image build would create the known_hosts file with incorrect ownership, this is legacy and no longer needed so it was removed.
```

Comment:
Let's remove the bullet, now only including items that impact end consumers of the library